cmd/tetra/common/client.go

```diff
@@ -73,7 +72,7 @@ func connect(ctx context.Context) (*grpc.ClientConn, string, error) {
 	}
 	if conn == nil {
 		// Try the server-address prameter
-		serverAddr = viper.GetString(KeyServerAddress)
+		serverAddr = ServerAddress
```

Comment:
just as a note we could remove this useless variable but I'm okay to merge this as is! :)